app/client/src/components/pages/MonitoringLocation.js

```diff
@@ -1358,7 +1378,9 @@ function CharacteristicChartSection({ charcName, charcsStatus, records }) {
                 />
               </span>
               <span className="radio-container">
-                <label>Scale Type:</label>
+                <span css={screenLabelStyles}>
+                  <GlossaryTerm term="Scale Type">Scale Type</GlossaryTerm>:
```

Comment:
I noticed that the glossary definition for this says that it can be found on the "Monitoring Report page". This is how the link from the Community Monitoring tab reads, but the title for the page is "Monitoring Location Details". We should double-check this in the future.

Reply:
Thanks for pointing these out. I'll send an email to Kiki, since she has to make changes to the definitions and based on what she says we can update the HMW code appropriately.

Reply:
I'm going to go ahead and merge this in and we can make an update later if needed.